addons/ccm-aws/ccm-aws.yaml

```diff
@@ -164,6 +164,8 @@ spec:
             - --cloud-provider=aws
             - --use-service-account-credentials=false
             - --configure-cloud-routes=false
+            - --allocate-node-cidrs=true
+            - --cluster-cidr={{ .Config.ClusterNetwork.PodSubnet }}
```

Comment:
Isn't this going to conflict with kube-controller-manager? I remember we had such issue with Azure in the past (cc @rastislavs).

Reply:
@PratikDeoghare @rastislavs This is still unanswered. ^^

Reply:
Hmm not sure. Let me check some things.